tox.ini

```diff
@@ -39,7 +39,10 @@ commands =
 changedir = {toxinidir}/docs
 whitelist_externals = htmlproofer
                       gem
-commands =
+
+commands_pre =
     gem install html-proofer
-    sphinx-build -b html -d {envtmpdir}/doctrees . {envtmpdir}/html
-    htmlproofer {envtmpdir}/html --alt-ignore '/.*/' --file-ignore '/2013/','/2014'/,'/2015/' --url-ignore '/video/' --allow-hash-href
+
+commands =
+    sphinx-build -b dirhtml -d {envtmpdir}/doctrees . {envtmpdir}/html
+    htmlproofer {envtmpdir}/html --alt-ignore '/.*/' --file-ignore -file-ignore "/2013/,/2014/,/2015/," --url-ignore '/2015/,/video/' --allow-hash-href
```

Comment:
`--file-ignore -file-ignore` feels like we're probably doing something wrong.

Reply:
gah, thanks!